docs/data-ai/train/train-tflite.md

```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: "Train TFlite model"
```

Comment:
out of diff nit: The internet seems to indicate it should be this way throughout (could search-replace)
```suggestion
linkTitle: "Train TFLite model"
```